File: src/test/java/com/drpicox/game/testPost/reader/PostSubsection.java
package com.drpicox.game.testPost.reader;

import java.util.ArrayList;
import java.util.List;

public class PostSubsection {
    private final List<PostLine> lines = new ArrayList<>();
    private final String name;

    public PostSubsection(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void addLine(PostLine line) {
        lines.add(line);
    }

    public List<PostLine> getLines() {
        return lines;
    }

    public PostLine getFirstLine() {
        return lines.get(0);
    }

}

File: src/www/__test__/testPost/reader/PostSubsection.ts
import { PostLine } from "./PostLine";

export class PostSubsection {
  private lines: PostLine[] = [];

  constructor(private name: string) {}

  addLine(line: PostLine) {
    this.lines.push(line);
  }

  getLines(): PostLine[] {
    return this.lines;
  }

  getFirstLine(): PostLine {
    return this.lines[0];
  }
}
